sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -114,6 +117,7 @@ public class DaprClientHttp extends AbstractDaprClient {
     this.client = client;
     this.isObjectSerializerDefault = objectSerializer.getClass() == DefaultObjectSerializer.class;
     this.isStateSerializerDefault = stateSerializer.getClass() == DefaultObjectSerializer.class;
+    System.out.println("NOTE: DaprClientHttp is deprecated and will be removed in a future release.");
```

Comment:
NIT, but it might be more noticeable if we change "NOTE" to "deprecation ALERT" or something to the effect that holds attention as something that shouldn't be ignored as a warning